highlight.js

```diff
@@ -123,64 +100,60 @@
     tern.defineQueryType('highlight', {
         run: function (server, query) {
             var messages = {};
-
-            server.files.forEach(function (file) {
-                var fileMessages = messages[file.name] = [];
-
-                var collect = function (node, type) {
-                    addMessage(fileMessages, node, type);
-                };
-
-                // Walk the error-tolerant parser AST
-                var visitors = makeVisitors(server, query, file, collect);
-                walk.simple(file.ast, visitors, scopeVisitor, file.scope);
-
-                // Re-parse to collect keyword tokens, comments, warnings and errors
-                try {
-                    var ast = acorn.parse(file.text, {
-                        ecmaVersion: 6,
-
-                        forbidReserved: 'everywhere',
-
-                        onToken: function (token) {
-                            token.type.keyword && collect({
-                                type: 'Keyword',
-                                start: token.start,
-                                end: token.end
-                            });
-                        },
-
-                        onComment: function (isBlock, text, start, end) {
-                            collect({
-                                type: 'Comment',
-                                start: start,
-                                end: end
-                            });
-                        },
-
-                        onInsertedSemicolon: function (pos) {
-                            collect({
-                                type: 'Warning',
-                                start: Math.max(0, pos - 1),
-                                end: pos,
-                                message: 'Missing semicolon'
-                            });
-                        },
-
-                        onTrailingComma: function (pos) {
-                            collect({
-                                type: 'Warning',
-                                start: pos,
-                                end: pos + 1,
-                                message: 'Trailing comma'
-                            });
-                        }
-                    });
-                } catch (err) {
-                    addError(fileMessages, err);
-                }
-            });
-
+            var fileMessages = messages["highlight"] = [];
+
+            var text = query.text, ast, collect = function (node, type) {
```

Comment:
I prefer having a separate `var` for each variable. If you could conform to this style, it would be cool.

`var text = query.text, ast, collect = function (node, type) {`
→
```
var text = query.text;
var collect = function (node, type) {
```